test/test_staticarrays.jl

```diff
@@ -1,11 +1,13 @@
 using StaticArrays
 
-obj = StaticArrays.@SMatrix [1 2; 3 4]
-l = @lens _[2,1]
-@test get(l, obj) == 3
-@test set(l, obj, 5) == StaticArrays.@SMatrix [1 2; 5 4]
-@test setindex(obj, 5, 2, 1) == StaticArrays.@SMatrix [1 2; 5 4]
-
-v = @SVector [1,2,3]
-@test (@set v[1] = 10) === @SVector [10,2,3]
-@test_broken (@set v[1] = π) === @SVector [π,2,3]
+let
```

Comment:
FYI I usually wrap each test file in a `module`.  Alternatively, you can wrap the whole test block with `@testset begin ... end`.  There is some discussion about this here https://github.com/JuliaDiffEq/DifferentialEquations.jl/issues/319

Reply:
I guess I wasn't clear.  If you put tests in `@testset begin ... end` *inside* `test/test_staticarrays.jl` file then it creates a namespace.  But `@testset begin include("test/test_staticarrays.jl") end` doesn't, as you observed.

Reply:
I like your idea of wrapping every file in a module. I think I will do that in another PR.